### ISSUE TITLE
PLP indicators disrupt container outlines in Firefox

### ISSUE THREAD

**Issue by tomgreenfield** [adaptlearning/adapt_framework#2619]
### Subject of the issue/enhancement/features
Container outline bounds are increased in height in Firefox due to the use of inline-block and whitespace between sibling elements.

### Your environment
* Framework 5.1
* Firefox 72.0.1
* Windows 10

### Steps to reproduce
1. Go to https://adaptlearning.github.io/v5demo/#/id/co-10
1. Run the following in the console:
```js
$(".c-60").css({ outline: "1px solid black", backgroundColor: "grey" });
```

### Expected behaviour
Top of container outline should be equal to its actual top.

### Actual behaviour
Container outline is pushed several pixels higher.

### Screenshots (if you can)
![plp](https://user-images.githubusercontent.com/922987/72059453-f2b53a80-32c9-11ea-8a4e-2be7fe22609f.png)

**Comment by akshay-bhavsar** [comment]
@tomgreenfield ,

There are two ways to solve this issue.
one could be Use of border property instead of using outline,
else we can remove "line-height" property of  "component__title" class, but that causes little disruption in height of component title text.

what do you think?

**Comment by tomgreenfield** [comment]
Hi @akshay-bhavsar, the proper way to fix this is to simply omit the whitespace in the component template:
https://github.com/adaptlearning/adapt_framework/blob/5b6460abe14a9908afa395bb947be4f7c7162fa2/src/core/templates/partials/component.hbs#L13
becomes:
```hbs
      {{~/unless~}}
```

**Comment by akshay-bhavsar** [comment]
works well!!

I will Raise PR for the same.

**Comment by akshay-bhavsar** [comment]
PR: https://github.com/adaptlearning/adapt_framework/pull/2666

**Comment by moloko** [comment]
fixed via 5e39624c2fd7ce016cf3fb4869146572da2e530e

**Comment by moloko** [comment]
actually - does this not also need applying to article & block titles as well?
